docs/src/content/docs/getting-started/configuration.mdx

```diff
@@ -134,20 +165,26 @@ Here's an example `workflows` configuration:
 
```

Comment:
The "Workflows" section features an advanced example. It would benefit from a simpler introductory example before exploring customization.